Library/Formula/mailhog.rb

```diff
@@ -0,0 +1,171 @@
+    url "https://github.com/mailhog/MailHog-Server.git",
+      :revision => "1dfb973b724a309c18474fe032b1458c0d8654fc"
+  end
+
+  go_resource "github.com/mailhog/MailHog-UI" do
+    url "https://github.com/mailhog/MailHog-UI.git",
+      :revision => "ebdfa0f91fac40169ca376953d7c13f9d83cd01a"
+  end
+
+  go_resource "github.com/mailhog/MailHog" do
+    url "https://github.com/mailhog/MailHog.git",
+      :revision => "91a9d8afe119dc96a6c9acfa94c997765c824908"
+  end
+
+  go_resource "github.com/ogier/pflag" do
+    url "https://github.com/ogier/pflag.git",
+      :revision => "6f7159c3154e7cd4ab30f6cc9c58fa3fd0f22325"
+  end
+
+  go_resource "github.com/t-k/fluent-logger-golang" do
+    url "https://github.com/t-k/fluent-logger-golang.git",
+      :revision => "0f8ec08f2057a61574b6943e75045fffbeae894e"
+  end
+
+  go_resource "github.com/tinylib/msgp" do
+    url "https://github.com/tinylib/msgp.git",
+      :revision => "c46fbee0675f25770e5f340dc9bc22bf4a108ad4"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://github.com/golang/crypto.git",
+      :revision => "c8b9e6388ef638d5a8a9d865c634befdc46a6784"
+  end
+
+  go_resource "github.com/philhofer/fwd" do
+    url "https://github.com/philhofer/fwd.git",
+      :revision => "8fd9a4b75098b2125fe442e48a3ffbf738254e13"
+  end
+
+  # Use file resource instead of revision until https://github.com/Homebrew/homebrew/issues/44828
+  # is resolved.
+  go_resource "labix.org/v2/mgo" do
+    url "https://github.com/go-mgo/mgo/archive/2e26580ebcca4eca6533535f2dd062cd6bff44ac.tar.gz"
+    sha256 "9ef389ccc5410e9cf54073b8db048739577c5ed8f2491cfd35f7e3a695082503"
+  end
+
+  def install
+    mkdir_p "#{buildpath}/src/github.com/mailhog/"
+    ln_s buildpath, "#{buildpath}/src/github.com/mailhog/MailHog"
+
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "MailHog"
+    bin.install "MailHog"
+  end
+
+  def caveats; <<-EOS.undent
+    To use MailHog as a sendmail replacement for PHP, add the
+    following line to php.ini file:
+
+      sendmail_path = #{HOMEBREW_PREFIX}/bin/MailHog sendmail test@test
+    EOS
+  end
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+      <key>KeepAlive</key>
+      <true/>
+      <key>Label</key>
+      <string>#{plist_name}</string>
+      <key>ProgramArguments</key>
+      <array>
+        <string>#{HOMEBREW_PREFIX}/bin/MailHog</string>
+      </array>
+      <key>RunAtLoad</key>
+      <true/>
+      <key>StandardErrorPath</key>
+      <string>#{var}/log/mailhog.log</string>
+      <key>StandardOutPath</key>
+      <string>#{var}/log/mailhog.log</string>
+    </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    output = shell_output("#{bin}/MailHog -h 2>&1 | head -n 1")
```

Comment:
Valid point. Given that `MailHog` outputs debug messages when run directly, test should check against that.